stress/common/src/main/java/alluxio/stress/worker/UfsIOParameters.java

```diff
@@ -23,19 +23,28 @@
  * Parameters used in the UFS I/O throughput test.
  * */
 public class UfsIOParameters extends Parameters {
+  public static final String USE_UFS_CONF = "--use-ufs-conf";
+  public static final String CONF = "--conf";
+  public static final String PATH = "--path";
+
   @Parameter(names = {"--threads"}, description = "the number of threads to use")
   public int mThreads = 4;
 
   @Parameter(names = {"--io-size"},
           description = "size of data to write and then read for each thread")
   public String mDataSize = "4G";
 
-  @Parameter(names = {"--path"},
-          description = "the Alluxio directory to write temporary data in",
+  @Parameter(names = {PATH},
+          description = "the Ufs Path to write temporary data in",
           required = true)
   public String mPath;
 
-  @DynamicParameter(names = "--conf",
+  @Parameter(names = {USE_UFS_CONF},
+      description = "If true, attempt to load the ufs configuration from an existing mount point "
+          + "to read/write to the base path")
```

Comment:
If this is set to `true`, does the existing `--conf` get ignored, or are they merged? Which takes priority? Can we add this info to the description?

Reply:
done